tests/common/datasets.py

```diff
@@ -31,6 +33,22 @@ def __getitem__(self, index: int):
         return self.x[index], self.y[index]
 
 
+def get_random_classification_dataloader(
```

Comment:
On second thought, I would resist the urge to refactor the dataloader creation to some common file, but rather add dist.get_sampler below everywhere. It's more verbose, but also more readable and communicates the minimal set of argumenst needed to create the dataloader for the test; these rarely change so non DRY is what I would suggest.